api/src/opentrons/protocol_api/core/engine/load_labware_params.py

```diff
@@ -81,6 +84,30 @@
     "usascientific_96_wellplate_2.4ml_deep": 2,
 }
 
+_APILEVEL_2_24_OT_DEFAULT_VERSIONS: dict[str, int] = deepcopy(
+    _APILEVEL_2_23_OT_DEFAULT_VERSIONS
+)
+_APILEVEL_2_24_OT_DEFAULT_VERSIONS.update(
+    {
+        "appliedbiosystemsmicroamp_384_wellplate_40ul": 3,
+        "biorad_384_wellplate_50ul": 4,
+        "biorad_96_wellplate_200ul_pcr": 4,
+        "corning_12_wellplate_6.9ml_flat": 4,
+        "corning_24_wellplate_3.4ml_flat": 4,
+        "corning_48_wellplate_1.6ml_flat": 5,
+        "corning_6_wellplate_16.8ml_flat": 4,
+        "corning_96_wellplate_360ul_flat": 4,
+        "nest_96_wellplate_100ul_pcr_full_skirt": 4,
+        "nest_96_wellplate_200ul_flat": 4,
+        "nest_96_wellplate_2ml_deep": 4,
+        "thermoscientificnunc_96_wellplate_1300ul": 3,
```

Comment:
I think we're missing `opentrons_96_wellplate_200ul_pcr_full_skirt`.

```suggestion
        "nest_96_wellplate_2ml_deep": 4,
        "opentrons_96_wellplate_200ul_pcr_full_skirt": 4,
        "thermoscientificnunc_96_wellplate_1300ul": 3,
```

Reply:
Actually wait, are we? That file still doesn't have any changes except for `version`—is it meant to?

Reply:
Yeah, it is. The other pull request will update some of these values as hw is doing measurements.
My pr establishes the "default labware" changes we need.